utils/determinants_tools.py

```diff
@@ -153,13 +156,37 @@ def csf_to_string(csf, bit_kind_size, size):
             import sys
             sys.exit(1)
 
-    with h5py.File(args.h5path, 'r') as f:
+    open_mode = 'r'
+    if args.transform:
+        open_mode = 'a'
+    with h5py.File(args.h5path, open_mode) as f:
+        alpha_group = 'MultiDet/CI_Alpha'
+        if alpha_group not in f:
+            alpha_group = 'MultiDet/CI_0'
+        beta_group = 'MultiDet/CI_Beta'
+        if beta_group not in f:
+            beta_group = 'MultiDet/CI_1'
 
-        bit_kind_size = f['MultiDet/CI_Alpha'].dtype.itemsize * 8
+        bit_kind_size = f[alpha_group].dtype.itemsize * 8
+        print(f[alpha_group].dtype)
 
         # We sanitize the determinants. Aka we transform any negative integers, into the proper unsigned one.
-        i_alpha= (sanitize(d,bit_kind_size) for d in f['MultiDet/CI_Alpha'])
-        i_beta= (sanitize(d,bit_kind_size) for d in f['MultiDet/CI_Beta'])
+        i_alpha= (sanitize(d,bit_kind_size) for d in f[alpha_group])
+        i_beta= (sanitize(d,bit_kind_size) for d in f[beta_group])
+        if args.transform:
+            alpha_dets = np.array([i for i in i_alpha], dtype=np.uint)
```

Comment:
Should this be [np.uint64](https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.uint64)? (I think on many/most platforms uint and uint64 will be the same, but if we always want uint64 then it might save some future headaches if that's specified here)

Reply:
Fixed!